README.md

```diff
@@ -12,24 +12,67 @@ See [Jenkins Distributed builds](https://wiki.jenkins-ci.org/display/JENKINS/Dis
 
 ## Running
 
+### Running with the SSH Build Agents plugin
+
 To run a Docker container
 
 ```bash
-docker run jenkins/ssh-agent "<public key>"
+docker run -d --rm --name=agent -p 4444:22 -e "JENKINS_AGENT_SSH_PUBKEY=<public_key>" jenkins/ssh-agent
```

Comment:
nitpick: either use long or short flags for all

Reply:
I'm sorry, but I don't know the long flag equivalent for [`-e`](https://docs.docker.com/engine/reference/run/#env-environment-variables), [`-d`](https://docs.docker.com/engine/reference/run/#detached--d) and [`-p`](https://docs.docker.com/engine/reference/run/#expose-incoming-ports).
I don't know either the short flag for [`--rm`](https://docs.docker.com/engine/reference/run/#clean-up---rm)  and [`--name`](https://docs.docker.com/engine/reference/run/#name---name). 😨

Reply:
I'll do the modification, thanks for the tip. ❤️